conftest.py

```diff
@@ -75,6 +75,40 @@ def fix_missing_hostname():
             logger.error("Failed to add hostname")
 
 
+@pytest.fixture(scope="session", autouse=True)
+@pytest.mark.uncollectif(lambda: version.current_version() < "5.6")
+def patch_appliance():
+    """ Patch appliance over SSH"""
+    logger.info('Patching appliance')
+    # (local_path, remote_path, md5/None) trio
+    patch_args = (
+        (str(patches_path.join('miq_application.js.diff')),
+         '/var/www/miq/vmdb/app/assets/javascripts/miq_application.js',
+         None),
+        (str(patches_path.join('autofocus.js.diff')),
+         '/var/www/miq/vmdb/app/assets/javascripts/directives/autofocus.js',
+         'f5ce9fa129d1662e6fe6f7c213458227'),
+    )
+    patched_anything = False
+    ssh_client = store.current_appliance.ssh_client
+    for local_path, remote_path, md5 in patch_args:
+        res = ssh_client.patch_file(local_path, remote_path, md5)
```

Comment:
This maybe a little contrived, but I would think having the patch method in ```utils.appliance``` would be nicer, it's just convenience in one way, but........we also need the ability to patch a "new" appliance, such as one that is going to be used in a context on the appliance stack. I would propose adding the ```patch_file``` method to the ```utils.appliance.IPAppliance``` and then adding a new method ```utils.appliance.IPAppliance.path_with_miqqe``` too, so that we can easily call the method, not just through conftest. Then conftest just calls that method on ```store.current_appliance```.